src/reducers/listUsersReducer.js

```diff
@@ -0,0 +1,18 @@
+import { RECV_DATA } from '../actions/ActionTypes';
+
+export default function listUsers(state=[],action){
```

Comment:
Why using state = []. I can not see any use of it. Are you clear about this use case of state here? You are always returning a new object. This is wrong approach. You need to merge the values using spread operators or Use object.assign.